### PR TITLE
Update drop docs

### PR DESCRIPTION
**Pull request by RyanMacG**
Mostly covering drops linked to authors/blog posts but also has availability day included

### CODE REVIEW

File: docs/reference/objects/accommodation/availabilty_day.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ When using an `availability_day` object you have access to the following attribu
 
 # availability_day.prices
 
-Returns an array of `variant_prices` for a day.
+Returns an array of [variant_prices]({% docs/reference/objects/product/variant_price.md %}) for a day.
```
Comment (ianmooney) [inline docs/reference/objects/accommodation/availabilty_day.md:11]: This is throwing an error for me. You need to include the `link` tag, e.g.

```
[variant_prices]({% link docs/reference/objects/product/variant_price.md %})
```

Also, I think you need to rebase with the latest, as it's complaining that `variant_price.md`.

```
Could not find document 'docs/reference/objects/product/variant_price.md' in tag 'link'. (ArgumentError)
```

You can use `bundle exec jekyll serve` to run it locally to check it all works properly.

Comment (RyanMacG) [reply docs/reference/objects/accommodation/availabilty_day.md:11]: Ah I think I messed this up in my last rebase, good spot!

Comment (RyanMacG) [reply docs/reference/objects/accommodation/availabilty_day.md]: Hmm the variant prices are still missing so might need to try a new branch and cherry pick

Comment (RyanMacG) [reply docs/reference/objects/accommodation/availabilty_day.md]: 🤦 we don't have the variant price drop in so I don't know what I was linking to previously...

**Comment by RyanMacG** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)

File: docs/reference/objects/blog_post.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Returns an [author]({% link docs/reference/objects/author.md %}) object for the blog's author.
+
+# blog_post.next_post
+
+Returns a `blog_post` object for the next post for the blog
+
+# blog_post.previous_post
+
+Returns a `blog_post` object for the next post for the blog
```
Comment (ianmooney) [inline docs/reference/objects/blog_post.md:27]: This should say _previous_ post.

File: docs/reference/objects/blog_post.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Blog Post
+parent: Objects
+---
+
+When using a `blog_post` object you have access to the following attributes.
+
+# blog_post.url
```
Comment (ianmooney) [inline docs/reference/objects/blog_post.md:9]: Can you make these methods alphabetical as well?

File: Gemfile [hunk 4]
```diff
@@ -26,3 +26,5 @@ end
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
+
+gem "webrick", "~> 1.7"
```
Comment (ianmooney) [inline Gemfile:30]: What was the reason for adding this gem?

**Review by ianmooney — COMMENTED** [review]
Nice work, Ryan 👍  Left a few comments in there.

File: Gemfile (hunk 4 above)
Comment (RyanMacG) [reply Gemfile:30]: https://easolhq.slack.com/archives/C03KMEBLYJ0/p1662111512702309

File: docs/reference/objects/blog_post.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Blog Post
+parent: Objects
+---
+
+When using a `blog_post` object you have access to the following attributes.
+
```
[the remark this answers is not in the document]
Comment (RyanMacG) [reply docs/reference/objects/blog_post.md:8]: @ianmooney I've updated this, could you take a quick look at the type one because the documented behaviour is a bit odd and a sense check would be great.

File: docs/reference/objects/accommodation/availabilty_day.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+layout: default
+title: Availability Day
+parent: Accommodation
+---
+
+When using an `availability_day` object you have access to the following attributes.
+
+# availability_day.available
+
+Returns true if the Accommodation is marked as available for a day.
```
Comment (ianmooney) [inline docs/reference/objects/accommodation/availabilty_day.md:11]: Can we lowercase "accommodation" and perhaps reword this to "is marked as available for this date"

File: docs/reference/objects/accommodation/availabilty_day.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+# availability_day.date
+
+Returns the date of the `availability_day`.
+
+# availability_day.prices
+
+Returns an array of [price]({% link docs/reference/objects/product/price.md %}) objects for a day.
```
Comment (ianmooney) [inline docs/reference/objects/accommodation/availabilty_day.md:19]: Same here, perhaps "for this date" ?

File: docs/reference/objects/accommodation/availabilty_day.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+When using an `availability_day` object you have access to the following attributes.
+
+# availability_day.available
+
+Returns true if the Accommodation is marked as available for a day.
+
+# availability_day.date
+
+Returns the date of the `availability_day`.
```
Comment (ianmooney) [inline docs/reference/objects/accommodation/availabilty_day.md:15]: How about the following:

```
The date as a timestamp. This can then be used in conjunction with Liquid's [ built in filters ](https://shopify.github.io/liquid/filters/date/).
```

Comment (ianmooney) [reply docs/reference/objects/accommodation/availabilty_day.md:15]: FYI - I've been avoiding using "Returns ..." at the start of every description and just stating what it is instead. e.g "The date ..." vs "Returns the date ...".

I think this reads better and is less repetitive.

File: docs/reference/objects/blog_post.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+Returns a `blog_post` object for the next post for the blog.
+
+# blog_post.previous_post
+
+Returns a `blog_post` object for the previous post for the blog.
+
+# blog_post.published_at
+
+Returns the date and time the blog post was published.
```
Comment (ianmooney) [inline docs/reference/objects/blog_post.md]: How about "Returns the time the blog post was published as a timestamp.". Just to be clear it doesn't return two things.

**Review by ianmooney — APPROVED** [review]
Looks good to me!

File: docs/reference/objects/blog_post.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+A `blog_post` object for the next post for the blog.
+
+# blog_post.previous_post
+
+A `blog_post` object for the previous post for the blog.
+
+# blog_post.published_at
+
+The date the blog was published as a timestamp. This can then be used in conjunction with Liquid's [ built in filters ](https://shopify.github.io/liquid/filters/date/).
```
Comment (geoffreymm) [inline docs/reference/objects/blog_post.md:31]: I think we've missed the `post` after `blog`

**Comment by ianmooney** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)
